profiles/models.py

```diff
@@ -29,6 +29,30 @@ def get_available_name(self, name, max_length=None):
         return name
 
 
+@reversion.register()
+class LegalRelationship(models.Model):
+    representative = models.ForeignKey(  # "parent"
+        "Profile", related_name="representatives", on_delete=models.CASCADE
+    )
+    representee = models.ForeignKey(  # "child"
+        "Profile", related_name="representees", on_delete=models.CASCADE
+    )
+    type = models.CharField(  # ATM only "custodianship"
+        max_length=30,
+        choices=settings.REPRESENTATION_TYPE,
+        default=settings.REPRESENTATION_TYPE[0][0],
+    )
+    confirmation_degree = models.CharField(
+        max_length=30,
+        choices=settings.REPRESENTATIVE_CONFIRMATION_DEGREE,
+        default=settings.REPRESENTATIVE_CONFIRMATION_DEGREE[0][0],
+    )
+    expiration = models.DateField(blank=True, null=True)
```

Comment:
does it need some default value or no?

Reply:
I suspect that this information is something that comes from the suomi.fi puolesta-asiointi service when the relationship is confirmed by VRK data.

Reply:
ok, we'll handle it later 👍